### PR TITLE
docs: Document OpenStack platform env vars.

### PR DESCRIPTION
**Pull request by russellb**

### CODE REVIEW

File: docs/user/environment-variables.md [hunk 1]
```diff
@@ -36,10 +36,15 @@ The installer accepts a number of environment variable that allow the interactiv
 
 ## Platform-Specific
 
+### AWS
```
Comment (wking) [inline docs/user/environment-variables.md]: +1 to these platform headers.

File: docs/user/environment-variables.md [hunk 2]
```diff
@@ -49,3 +54,14 @@ The installer accepts a number of environment variable that allow the interactiv
 
     **Warning**: you should only set this if you're testing RHCOS releases.
     Most users should allow the installer to choose the OS image.
+
+### OpenStack
+
+* `OPENSHIFT_INSTALL_OPENSTACK_CLOUD`:
+     The OpenStack cloud name from clouds.yaml.
+* `OPENSHIFT_INSTALL_OPENSTACK_REGION`:
+     The OpenStack region to be used for installation. For example, “regionOne”.
```
Comment (wking) [inline docs/user/environment-variables.md]: nit: I prefer backticks (e.g. `regionOne`) to curly quotes for literals like this (and also for paths like `clouds.yaml` above).

Comment (russellb) [reply docs/user/environment-variables.md]: Ah, good suggestion, I'll update this in a moment.

**Comment by akostadinov** [comment]
Would be useful to also document 
```
export OS_CLIENT_CONFIG_FILE=/home/jenkins/clouds.yaml
```
That you can set location of `clouds.yaml`. it is from `pkg/asset/installconfig/openstack/openstack.go`. Perhaps list all the locations `clouds.yaml` would be read from. FYI current directory may not work though. Because installer finds it, but then terraform fails. Maybe the terraform binary is executed in another dir so there might be a bug.

**Comment by sferich888** [comment]
We should also link to https://docs.openstack.org/python-openstackclient/pike/configuration/index.html#configuration-files and explain how to get or generate such a file.

**Comment by crawford** [comment]
We are in the process of removing support for environment variables. They were only meant to be a temporary workaround for CI and (as predicted) have ended up confusing users. As such, I'm going to close this out.

/close

**Comment by wking** [comment]
Is the current alternative robust godocs?  E.g. https://godoc.org/github.com/openshift/installer/pkg/types#InstallConfig (which could use some fleshing out).  Or do we want YAML-flavored docs without the Go distractions (for users anyway) up somewhere?

**Comment by crawford** [comment]
Eventually, we will want some docs for the install configs. For now, I'm fine asking users to run `create install-config` and then modifying that.

**Pull request closed without merging**